code/game/objects/random/random.dm

```diff
@@ -1102,12 +1103,11 @@
 	icon_state = "breacher_rig"
 	spawnlist = list(
 		/obj/item/weapon/rig/ce/equipped = 1,
-		/obj/item/weapon/rig/ert/janitor = 0.1,
-		/obj/item/weapon/rig/eva/equipped = 0.5,
-		/obj/item/weapon/rig/hazard = 1,
-		/obj/item/weapon/rig/hazmat = 1,
-		/obj/item/weapon/rig/unathi = 1,
-		/obj/item/weapon/rig/vaurca/minimal = 0.1
+		/obj/item/weapon/rig/eva/equipped = 1,
```

Comment:
I'm not entirely comfortable with giving a bunch of fully equipped rigs on the vault tbh.